serving/docker/lmi.Dockerfile

```diff
@@ -28,7 +28,7 @@ ARG optimum_version=1.19.1
 ARG auto_gptq_version=0.7.1
 ARG datasets_version=2.19.0
 # LMI-Dist Deps
-ARG vllm_wheel="https://github.com/vllm-project/vllm/releases/download/v0.4.0/vllm-0.4.0-cp310-cp310-manylinux1_x86_64.whl"
+ARG vllm_wheel="https://github.com/vllm-project/vllm/releases/download/v0.4.0.post1/vllm-0.4.0.post1-cp310-cp310-manylinux1_x86_64.whl"
```

Comment:
We rebased with vllm 0.4.0, did not test with 0.4.0post1. So we should keep it 0.4.0 for now and followup with 0.4.1